use-cases/telltale.md

```diff
@@ -0,0 +1,179 @@
+# System definition
```

Comment:
Add licence string

Reply:
Hi @jochen-kall I will ignore this comment. By reading through the docs as we have the LICENSE file in the top level directory of this repository, this may be sufficient.